roles/openshift_node/tasks/upgrade/rpm_upgrade.yml

```diff
@@ -10,6 +10,8 @@
   command: "{{ ansible_pkg_mgr }} install -y --downloadonly {{ openshift_node_upgrade_rpm_list | join(' ')}}"
   register: result
   until: result is succeeded
+  failed_when:
+    - result.stdout is search(".*No package .* available.*") or result is failed
```

Comment:
This syntax looks correct according to docs, but it's quite non-intuitive.  But, probably what we should use here.

Reply:
I agree, it is rather unusual.

And for clarity, the reason I used the extra `.*`s is because that `stdout` is multi-line and there are random `\n`s. But, after re-reading the docs, I don't think it's necessary (`match` requires a full string match, `search` does not), but it won't hurt anything either.